man/zypper-docker-lp.1.md

```diff
@@ -0,0 +1,41 @@
+% ZYPPER-DOCKER(1) zypper-docker User manuals
+% SUSE LLC.
+% SEPTEMBER 2015
+# NAME
+zypper\-docker list-patches \- List all the available patches.
+
+zypper\-docker list-patches-container \- List all the available patches for
+the given container.
+
+# SYNOPSIS
+**zypper-docker list-patches** [command options] IMAGE
+
+**zypper-docker list-patches-container** [command options] CONTAINER
+
+# DESCRIPTION
+The **list-patches** command lists all the patches that are available for the
+given openSUSE/SLES image. The provided image follows the same naming
+conventions as in Docker. To fetch which images are based on openSUSE or SLES,
+use the **images** command.
+
+The **list-patches-container** takes the container ID and lists the patches for
+the image in which the given container is based on.
```

Comment:
I think we should add something like: "Note well: the running container is not going to be affected, zypper-docker will spawn a new container based on the same image used by the running one".